packages/headless/src/features/commerce/sort/sort-slice.test.ts

```diff
@@ -95,7 +95,9 @@ describe('product-listing-sort-slice', () => {
     it('resets sort', () => {
       state.appliedSort = sort;
       state.availableSorts = [sort];
-      const finalState = sortReducer(state, action);
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const finalState = sortReducer(state, action({} as any));
```

Comment:
I'm not fond of this, as this IMO should not work/pass, as the action creators should :rage1: if you pass an empty obj, and thus the reducer should not be triggered.

But I reckon this is outside the scope of this PR.

Same for the pagination-slice ☝️